guides/common/modules/con_troubleshooting.adoc

```diff
@@ -1,57 +1,6 @@
 [id="Troubleshooting_{context}"]
 = Troubleshooting
 
-ifdef::foreman-el,katello[]
```

Comment:
I think in this PR this remain with an include. That way you'll after merging this PR there shouldn't be any diff in the resulting files.

Reply:
I have added this section to `con_troubleshooting-ruby.adoc`. Can we add the ifdefs in the assembly?

Reply:
I'm not sure what you mean. If I look at the PR I don't see you adding the section to `con_troubleshooting-ruby.adoc`.

Let me put it another way: for me the acceptance criteria would be that the generated diff (https://github.com/theforeman/foreman-documentation/pull/1832#issuecomment-1320079343) is empty, or at most some small changes to headers. Right now it can't be generated, but if it would I think you'd drop the DNF module troubleshooting would be gone from all documents.

Reply:
`con_troubleshooting-dnf-modules.adoc` is gone with the latest commit. It further got splitted into `troubleshooting-ruby` and `troubleshooting-postegresql`. 

In `troubleshooting-ruby`, I have added the section of `ifdef::foremanel-katello[]` (line 4) - upto `# dnf module reset ruby` (line 18).

Reply:
Perhaps you forgot to push the change, because I don't see it here.

Reply:
I am yet to push this change [comment](https://github.com/theforeman/foreman-documentation/pull/1832/files#r1029273061)

Reply:
> I am yet to push this change [comment](https://github.com/theforeman/foreman-documentation/pull/1832/files#r1029273061)

Pushed

Reply:
It's still unresolved. Please look at https://github.com/theforeman/foreman-documentation/pull/1832#issuecomment-1320079343. I expect that the comment there is that there is no change. That means you must insert include statements again. So I'd expect this to be present:
```asciidoc
ifdef::foreman-el,katello[]
[id="troubleshooting_dnf_modules"]
== DNF modules

include::con_troubleshooting-ruby.adoc[leveloffset+=1]
include::con_troubleshooting-postgresql.adoc[leveloffset+=1]
end::[]
```

Reply:
Addressed :+1: